sklearn/tests/test_docstring_parameters.py

```diff
@@ -0,0 +1,173 @@
+"""
+Test mismatches between docstring and method/function signature.
+"""
+
+# Authors: Joel Nothman <joel.nothman@gmail.com>
+# License: BSD 3 clause
+
+
+# TODO inspect for Cython (see sagenb.misc.sageinspect)
+from __future__ import print_function
+
+import os
+import inspect
+import imp
+import sys
+import sklearn
+import warnings
+from pkgutil import walk_packages
+from nose import SkipTest
+
+
+scikit_learn_path = os.getcwd().replace("/sklearn/tests", "")
+docscrape_path = 'doc/sphinxext/numpy_ext/docscrape.py'
+docscrape = imp.load_source('docscrape',
+                            os.path.join(scikit_learn_path, docscrape_path))
+
+# To capture warnings related to invalid section names in docs
+warnings.simplefilter('always')
```

Comment:
Okay! and can't as in shouldn't because of any possible side effects of the same... or because it won't have the intended effect?